processor/k8sattributesprocessor/doc.go

```diff
@@ -60,7 +60,7 @@
 //     - container.image.name
 //     - container.image.tag
 //   2. Container status attributes - in addition to pod identifier and `container.name` attribute, these attributes
```

Comment:
This comment seems incorrect if the container ID is being substituted with the restart count.

Reply:
Container ID is being fetched from k8s API and added to the list of attributes, only if the pod identified and two additional attributes provided `k8s.container.name` and `k8s.container.restart_count`. It actually must be `k8s.container.name` instead of `container.name` in the comment. Fixed. Otherwise the comment looks correct, may be not the best wording